Kudu.Core/Deployment/Generator/ExternalCommandBuilder.cs

```diff
@@ -129,7 +128,7 @@ public IList<string> GetPostBuildActionScripts()
             var defaultPath = Path.Combine(Environment.DeploymentToolsPath, PostDeploymentActions);
             if (!string.IsNullOrEmpty(System.Environment.GetEnvironmentVariable(SettingsKeys.PostDeploymentActionsDirectory)))
```

Comment:
If old logic only supported absolute file path, what caused the regression in #1894?